tensorflow/core/kernels/mlir_generated/build_defs.bzl

```diff
@@ -89,16 +89,18 @@ _gen_mlir_op_rule = rule(
     implementation = _gen_mlir_op_impl,
 )
 
-def _gen_mlir_op(op, type, platform, output_type):
+def _gen_mlir_op(op, name, type, platform, output_type):
     _gen_mlir_op_rule(
         name = "generate_{op}_{platform}_{type}_{output_type}_mlir".format(
```

Comment:
Is this uique w/o the name in it? If the name is not needed, why is it passed below?

Reply:
name is removed from function